explainability-core/src/main/java/org/kie/trustyai/explainability/local/tssaliency/TSSaliencyModelWrapper.java

```diff
@@ -0,0 +1,76 @@
+package org.kie.trustyai.explainability.local.tssaliency;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import org.kie.trustyai.explainability.model.*;
+
+/**
+ * A wrapper for a {@link PredictionProvider} that converts inputs between time-series vector format and
+ * list of features format
+ */
+public class TSSaliencyModelWrapper implements PredictionProvider {
+
+    private final PredictionProvider model;
+
+    public TSSaliencyModelWrapper(PredictionProvider model) {
+        this.model = model;
+    }
+
+    /**
+     * Convert predictions containing a single vector feature to a list of features.
+     * 
+     * @param inputs
+     * @return A list of {@link PredictionInput}
+     */
+    public static List<PredictionInput> featureVectorTofeatureList(List<PredictionInput> inputs) {
```

Comment:
Do you think this logic might be useful also in general? I would probably move the static methods in another utility class and here just keep the `predictAsync` impl.
Wdyt?

Reply:
Move to `TimeseriesUtils`.